src/main/java/seedu/planner/model/course/FocusArea.java

```diff
@@ -36,6 +36,10 @@
     private static boolean hasFocusArea(String focusArea) {
         focusArea = focusArea.toLowerCase();
         for (FocusArea fa : FocusArea.values()) {
+            // Do not allow the setting of an {@code UNKNOWN} focus area
```

Comment:
Sorry, can I ask you to revert changes to this file and Major.java? I am disabling some majors and focus areas in `setup` command and also doing a bit of refactoring. Having these changes in your PR means I will need to wait until this PR is merged.

Reply:
Ok I'll revert. I'll merge this PR after if everything's ok

Reply:
Thank you!